maven-plugin/src/it/default-configuration/invoker.properties

```diff
@@ -0,0 +1 @@
+invoker.goals=-e clean verify
```

Comment:
this file configures the goals to run for this specifis maven project

See http://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#invokerPropertiesFile